anitya/lib/backends/pagure.py

```diff
@@ -59,7 +61,7 @@ def get_versions(cls, project):
         try:
             req = cls.call_url(url)
         except Exception as err:  # pragma: no cover
-            print err
+            print(err)
```

Comment:
Yes, I think we could just drop the print, other backends aren't printing either